packages/aggregator/aggregator/aggregator.py

```diff
@@ -260,7 +257,8 @@ def process_batch_result(worker_results: WorkerResults):
             print(f"Error clearing user data: {e}")
 
         print("Creating and sending final report")
-        send_to_clients(
+        manager.send_to_user(
```

Comment:
Where is manager defined? I can't see it on my phone so am probably just missing it

Reply:
Wait it's at the bottom of the file - move it to the top and could be a good idea to move ConnectionManaged out to its own file tbh